testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -165,26 +164,24 @@ public void testInsert()
     }
 
     @Test
-    public void testDelete()
+    public void testRowLevelDelete()
```

Comment:
i'd suggest reverting this change.
`hasBehavior` should allow us to drive the reusable test without need to split it up.
the need to split up the test indicates that testing behaviors are not defined/plugged ideally yet